StripeFinancialConnections/StripeFinancialConnections/Source/Common/TestModeAutofillBannerView.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  TestModeAutofillBannerView.swift
+//  StripeFinancialConnections
+//
+//  Created by Mat Schmid on 2024-06-20.
+//
+
+@_spi(STP) import StripeCore
+@_spi(STP) import StripeUICore
+import UIKit
+
+class TestModeAutofillBannerView: UIView {
+
+    enum Context {
+        case account
+        case otp
+
+        var buttonLabel: String {
+            switch self {
+            case .account:
+                STPLocalizedString(
+                    "Use test account",
+                    "Label for a button shown to a user who is in test mode, which will give them the option to autofill mock account credentials."
+                )
+            case .otp:
+                STPLocalizedString(
+                    "Use test code",
+                    "Label for a button shown to a user who is in test mode, which will give them the option to autofill a mock one time passcode."
+                )
+            }
+        }
+    }
+
+    private let context: Context
+    private let didTapAutofill: () -> Void
+
+    // MARK: - Subviews
+
+    private lazy var messageLabel: UILabel = {
+        // Create icon as a text attachement.
+        let icon = Image.info
+            .makeImage(template: true)
+            .withTintColor(.attention300)
+        let textAttachment = NSTextAttachment(image: icon)
+
+        textAttachment.bounds = CGRect(
+            x: 0,
+            y: -3,
+            width: icon.size.width,
+            height: icon.size.height
+        )
+
+        let attributedString = NSMutableAttributedString()
+        attributedString.append(NSAttributedString(attachment: textAttachment))
+        // Add a spacer between the icon and the label in the form of a space.
+        attributedString.append(NSAttributedString(string: " "))
+        attributedString.append(NSAttributedString(string: STPLocalizedString(
+            "You're in test mode.",
+            "Message shown to a user who is in test mode, which will give them the option to autofill mock credentials."
+        )))
+
+        let label = UILabel()
```

Comment:
it shouldn't matter too much here because its a single line text wrapped with padding, but generally prefer `AttributedLabel` because that supports line heights together with `FinancialConnectionsFont` (`UILabel` usage in Financial Connections is likely very small if one searches it)